mods/common/languages/en.ftl

```diff
@@ -860,3 +860,21 @@ notification-desync-compare-logs = Out of sync in frame { $frame }.
 
 ## SupportPowerTimerWidget
 support-power-timer = { $player }'s { $support-power }: { $time }
+
+## WidgetUtils
+player-status =
+    { $player } { $win-state ->
+        [won] Won
+        [lost] Lost
+       *[other] {""}
+    } { $client-state ->
+        [disconnected] (Gone)
+       *[other] {""}
+    }
```

Comment:
this conflicts with https://github.com/projectfluent/fluent/wiki/Good-Practices-for-Developers

this logic should be inside c# and not ftl files

Reply:
I thought the word order might be different in other languages.